commit 3cbf3cd6eb97f6a3333fc98de429e5d857aa040d (refs/changes/57/757/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-10-21T00:01:46+00:00 (3 years ago)
    
    CNS-171 log rotation setup is incomplete
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/boot/setup.sh b/boot/setup.sh
index 731356f..3199515 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -72,6 +72,7 @@ svcadm enable cns-updater
 # add log rotation entries for services
 sdc_log_rotation_add cns-updater /var/svc/log/*cns-updater*.log 1g
 sdc_log_rotation_add cns-server /var/svc/log/*cns-server*.log 1g
+sdc_log_rotation_setup_end
 
 # All done, run boilerplate end-of-setup
 sdc_setup_complete
